README.md

```
@@ -1 +1,166 @@
+| 박소해 | 팀원 | 상품, 이벤트 |
+| 이지웅 | 팀원 | 주문, 결제 |
+| 임대일 | 팀원 | 회원, 쿠폰 |
+
+<br>
+
+## 프로젝트 구조
+
+<details>
+<summary></summary>
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)